assets/src/components/map.tsx

```diff
@@ -290,6 +292,55 @@ export const autoCenter = (
   }
 }
 
+export const drawerOffsetAutoCenter: UpdateMapFromPointsFn = (map, points) => {
```

Comment:
praise: I like this rename - `drawerOffsetAutoCenter` is very clear! 
question (non-blocking): does this need to be moved out of `mapDisplay?`

Reply:
I chose to move it out of `mapDisplay` to put it in the same place as the other `autoCenter` function, just because it seemed weird to have two functions that do very similar things in different places in the code. I'd be open to other suggestions for how to organize things, though.